week-2/F-logical-operators/exercise.js

```diff
@@ -27,10 +27,7 @@ var cssOrHtmlAbove5;
 console.log("Is Html knowledge above 5?", htmlLevelAbove5);
 console.log("Is CSS knowledge above 5?", cssLevelAbove5);
 console.log("Is Html And CSS knowledge above 5?", cssAndHtmlAbove5);
-console.log(
-  "Is either Html or CSS knowledge above 5?",
-  cssOrHtmlAbove5
-);
+console.log("Is either Html or CSS knowledge above 5?", cssOrHtmlAbove5);
```

Comment:
Did you get stuck here? This exercise is for combining boolean values using "and" and "or". These are written `&&` and `||` in Javascript. e.g.

```
var imHungry = true;
var imThirsty = false;

var imHungryAndThirsty = imHungry && imThirsty;
```